### ISSUE TITLE
TokenAuthentication and SessionAuthentication run side-by-side?

### ISSUE THREAD

**Issue by JeffreyZZ** [rrebase/knboard#64]
The backend enables both Django TokenAuthentication and SessionAuthentication authentication in settings. I observed that only SessionAuthentication is being used if I visit the page within browser as run the application on my local machine. But strangely, I also see authToken_token table has an entry updated.  Does this mean both TokenAuthentication and SessionAuthentication run side-by-side for a single request?

**Comment by zhan9san** [comment]
> It seems that TokenAuthentication is enabled but not being returned or used.

Yes, you're correct. IMHO, `rest_framework.authentication.TokenAuthentication` can be  removed.

The returned data format is described in [TokenSerializer](https://github.com/rrebase/knboard/blob/master/backend/accounts/serializers.py#L78-L80), and you can find it in [dj-rest-auth](https://github.com/rrebase/knboard/blob/master/backend/config/urls.py#L54)
